PerfNext/app/apis/benchmark_data.js

```diff
@@ -20,26 +20,12 @@ module.exports = function(app) {
     /*
     * Get all benchmark definitions by reading all files from data folder (root/config/benchmarks). 
     * This method appends all files to a single file (caches it) and then sends it to the caller.
-    * 
-    * Params: isSimple (boolean, default = false) - Specifies which set of benchmark definitions to send (see comments below for more details)
     */    
 
-    app.get('/api/getBenchmarks', function(req, res) {
-                
-        // We currently have 2 interfaces of dev, the simple interface and the full launcher.
-        // The full launcher was developed before BenchEngine, hence the data definitions are the same
-        // as old "perffarm". For BenchEngine, the benchmark definitions needed to made more atomic. For this reason,
-        // we differentiate between the benchmark definitions. If isSimple param is passed, it specifies that the consumer is interested in the new defiitions which are
-        // compatible with BenchEngine (This is currently used by "simple launcher"). The new data definitions are stored in data_simple
-        // and cached to benchmarks_simple_compiled.xml instead of benchmarks_compiled.xml.
-
-
-        //Define the data folder and cache file based on the isSimple flag
-        var dataFolder = req.query.isSimple ? 'data_simple' : 'data';
-        var compiledBenchmarksFile = req.query.isSimple ? 'benchmarks_simple_compiled.xml' : 'benchmarks_compiled.xml';
+    app.get('/api/getBenchmarks', function(req, res) {              
         
-        var benchmarkDataFolderPath = global.appRoot + `/config/benchmarks/${dataFolder}/`;
-        var compiledBenchmarksPath = global.appRoot + `/config/cache/${compiledBenchmarksFile}`;
+        var benchmarkDataFolderPath = global.appRoot + "/config/benchmarks/data_simple/";
+        var compiledBenchmarksPath = global.appRoot + "/config/cache/benchmarks_compiled.xml";
```

Comment:
According to the previous logic, should we use `benchmarks_simple_compiled.xml` here?

Reply:
That's right! According to the previous logic, we should use `benchmarks_simple_compiled.xml` but it doesn't really matter since it's just a name of an aggregated xml and we no longer care about simple or not simple. Hence, I preferred to go with the shorter name.